ruby/ql/lib/upgrades/3595c826de6db850f16b9da265a54dbf24dd3126/upgrade.properties

```diff
@@ -0,0 +1,2 @@
+description: <INSERT DESCRIPTION HERE>
+compatibility: full|backwards|partial|breaking
```

Comment:
You forgot to change this.

Reply:
Ah indeed. I changed it, but I regenerated the files because I had forgotten about the `numlines` table.

Reply:
> Why did the PR checks pass?

CI doesn't check the description or compatibility fields. It only checks the dbscheme and whether the upgrade queries can be compiled (there are none in this case). The "upgrade" is simply adding empty tables which can be achieved by doing nothing at all ;-)